O/OCCT/build_tarballs.jl

```diff
@@ -0,0 +1,117 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "OCCT"
+version = v"7.6.2"
+
+# Collection of sources required to build Open CASCADE Technology (OCCT)
+sources = [
+    FileSource("https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_6_2;sf=tgz",
```

Comment:
Might be easier to use a [`GitSource`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.GitSource), no?

Reply:
Yep I agree, resolved in b12f87e.